DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+		\hline
+  calibrated flux and associated error for the DIAsource as measured on the Visit image & FAIL & not present in the catalog \\
+		\hline
+\end{tabular}
+	\caption{Required DIASource attributes for AG-00-20 \label{tab:diasrc}}
+\end{table}
+
+
+\textit{Faint DIASources satisfiying additional criteria are stored: FAILED}
+
+No specification for the ``additional criteria'' necessary to store 
+sub-threshold DIASources are available at this time; 
+this is a failure of requirements rather than of the pipelines.
+
+A new JIRA ticket, DM-12967, has been created to capture this issue.
+
+\textit{Relevant derived quantities are provided in pre-computed columns: PASSED}
+
+Examination of the DIASource record shows two entries,
+\texttt{ip\_diffim\_PsfDipoleFlux\_chi2dof} and
+\texttt{ip\_diffim\_DipoleFit\_chi2dof}, which provide goodness of fit measures
+as required.  Color-color diagrams are not relevant for single-visit DIASource
+catalogs. The requirement does not specify an exhaustive list of relevant
+derived quantities and hence cannot be fully tested as written.
+
 \subsection{AG-00-25}
 
-\appendix
+\textit{Measurements are Presented in Flux Units: FAILED}
+
+No measurements (or aggregated measurements, such as means) 
+were contained in the the DIAObject table as implemented at the 
+time of the test.
+
+\textit{Each DIAObject record contains required attributes: FAILED}
+
+From DMS-REQ-0271, the required attributes include: a unique identifier, 
+identifiers of the nearest stars and galaxies in the Object catalog, 
+a probability that the DIAObject is the Object, and a set of DIAObject 
+properties.
+
+We verified that all DIAObjects had a unique identifier by comparing the 
+output of \texttt{select count(distinct(id)) from dia\_objects;} and
+\texttt{select count(id) from dia\_objects;} on the Level 1 database.  
+Both returned 546688 objects.
+
+The attributes linking to the Object catalog are out of scope for this test, as
+they require joint processing with DRP.
+
+By examining the schema of the implemented Level 1 database, we determined that only one DIAObject property is implemented at this time, a count of the number
+of DIASources matched to it.  However, all values in the column were 1, which 
+is not expected for a real astrophysical dataset.
+
+
+From DMS-REQ-0272, the required summary attributes include: 
+position, proper motion and parallax, magnitudes and errors per passband; 
+mean forced photometry flux and errors; variability measures; and flags.
+
+None of these attributes are contained in the Level 1 database 
+implemented as of this test.
+
+
+\textit{Each DIAObject record characterizes varability using DIASources contained in the required time interval: FAILED}
+
+The timeseries features to be computed on DIAObjects have not been finalized
+(cf.\ DM-11962), and a minimal subset has yet to be implemented.  Accordingly
+as of this writing there are no variability metrics computed on the DIAObjects.
```

Comment:
I'd probably say this couldn't be tested because it hasn't been implemented, hence it's a deviation rather than a failure.